test/spec.error.js

```diff
@@ -1,99 +1,126 @@
-var ErrorClass = require('../lib/error');
+'use strict';
 
-describe('Error', function () {
+const ErrorClass = require('../lib/error');
 
-    var req, res;
+describe('Error', () => {
+    let req, res;
 
-    beforeEach(function () {
-        req = request({
-            translate: sinon.stub().returnsArg(0)
+    describe('constructor', () => {
+        beforeEach(() => {
+            sinon.stub(ErrorClass.prototype, 'getMessage');
+            req = {};
+            res = {};
         });
-        res = response();
+
+        afterEach(() => {
+            ErrorClass.prototype.getMessage.restore();
+        });
+
+        it('should set key and options to the class', () => {
+            let options = { type: 'errortype' };
+            let error = new ErrorClass('key', options, req, res);
+            error.key.should.equal('key');
+            error.type.should.equal('errortype');
+        });
+
+        it('should allow being called with no options', () => {
+            let error = new ErrorClass('key');
+            error.message;
```

Comment:
I was originally going to runt he message getter and then check that getMessage had been called with the correct args, but that gets covered later. Removed